yarn.lock

```diff
@@ -1217,6 +1217,21 @@
   resolved "https://registry.yarnpkg.com/@rest-hooks/use-enhanced-reducer/-/use-enhanced-reducer-1.0.5.tgz#2e641e4962d98e5b161bcc5d8f5a4989df8d8911"
   integrity sha512-WZANpcS+FLNqzW409zzgOgdw6tdEKEJMLdGT6y6sZ9g6ZRriaZF0L+v7ybeVN6+W6t6p/EliNSBXHJhu6/pvTA==
 
+"@rx-store/core@^1.1.0":
+  version "1.1.0"
+  resolved "https://registry.yarnpkg.com/@rx-store/core/-/core-1.1.0.tgz#41c15ee1112db8a75c6b21c5ba88a3884efd7011"
+  integrity sha512-SJDv067uYWzV/BFxNMgQnLshn05dPe/edttTnRl3MQbFPoSrlITrxug1giEaFJLfeYUhPMHzBzuK2Ifh+qADSw==
+  dependencies:
+    debug "^4.1.1"
+
+"@rx-store/react@^1.1.0":
+  version "1.1.0"
+  resolved "https://registry.yarnpkg.com/@rx-store/react/-/react-1.1.0.tgz#121e0c1fc1cea56eb9a0bc130c092d79dc23ed1f"
+  integrity sha512-o2dtvJ3D/42wGlFl2AqJIF2/QJNq4A4Cu/SdCSGhiI3zse6hUSkyEY9dsI+ZfcfGd/DklYLIkirpxYT4ClSKWQ==
+  dependencies:
+    debug "^4.1.1"
+    react-dom "16.13.1"
```

Comment:
We'd like to test against react@experimental. Is there any reason you don't make this peer dependency? I don't thinks it's very important, though.

Reply:
It should be peer. I’ll fix this and ping once it’s fixed. Sorry